media/js/mozorg/contribute/signup.js

```diff
@@ -0,0 +1,21 @@
+$(function() {
```

Comment:
I guess not reallllllly but, perhaps safer to leave as is, else line two below might get an empty array if those elements does not exist in the DOM yet, right?